src/include/migraphx/op/reshape.hpp

```diff
@@ -26,6 +27,8 @@ struct reshape
         return pack(f(self.dims, "dims"));
     }
 
+    value attributes() const { return {{"std_shape", true}}; }
```

Comment:
Maybe the attribute should be called `require_std_shape`.

Reply:
fixed